bin/cron_script.sh

```diff
@@ -50,11 +50,11 @@ elif [ "$HOST_CLOUD" = 'azure' ]; then
   sed -i -e '/image_publisher/,+3d' build-azure.json
 
   # Searching and composing URL of to a custom VHD to use, i.e. the latest kubenow stable image
-  vhd_name=$(az storage blob list --account-name "$AZURE_STORAGE_ACCOUNT" --container-name "$AZURE_CONTAINER_NAME" --query [].name --output tsv | grep "kubenow-$CURRENT_VERSION" | grep '.vhd')
+  vhd_name=$(az storage blob list --account-name "$AZURE_STORAGE_ACCOUNT" --container-name "$AZURE_CONTAINER_NAME" --query [].name --output tsv | grep "kubenow-$CURRENT_VERSION[^-abcr]" | grep '.vhd')
```

Comment:
So here the [^-abcr] appended at the end of the grep is crucial, otherwise both stable, current and test images' URL are returned. This was making the next sed going nuts because of hidden newlines escape characters.